src/django/api/serializers.py

```diff
@@ -632,6 +640,62 @@ def get_affiliation_choices(self, claim):
     def get_certification_choices(self, claim):
         return FacilityClaim.CERTIFICATION_CHOICES
 
+    def get_product_type_choices(self, claim):
+        seeds = [
+            seed
+            for seed
+            in ProductType.objects.all().values_list('value', flat=True)
+            or []
+        ]
+
+        new_values, = FacilityClaim \
+            .objects \
+            .all() \
+            .values_list('facility_product_types', flat=True)
+
+        values = [
+            new_value
+            for new_value
+            in new_values or []
+        ]
+
+        union_of_seeds_and_values = list(set(values).union(seeds))
+        union_of_seeds_and_values.sort()
+
+        return [
+            (choice, choice)
+            for choice
+            in union_of_seeds_and_values
+        ]
```

Comment:
I revised how these serializers were working to make them return a list of tuples like `[("Accessories", "Accessories")]` etc. This matches how the other choice serializers work here which makes it easier to reuse the existing `mapDjangoChoiceTuples` stuff in the client.

Reply:
:+1: